libs/ui-lib/lib/ocm/hooks/_use-feature-detection.test.tsx

```diff
@@ -51,12 +51,7 @@ describe('use-feature-detection.ts', () => {
       ),
     );
 
-    expect(
-      isFeatureEnabled(
-        storeDay1.getState() as RootStateDay1 & FeatureFlagsState,
-        'ASSISTED_INSTALLER_PLATFORM_OCI',
-      ),
-    ).toBe(true);
+    expect(storeDay1.getState().featureFlags.data['ASSISTED_INSTALLER_PLATFORM_OCI']).toBe(true);
```

Comment:
Please revert and move elsewhere if necessary.
This code was using the `isFeatureEnabled` selector.
You can rewrite it like this in order to reuse it:
```
expect(isFeatureEnabled('ASSISTED_INSTALLER_PLATFORM_OCI')(storeDay1.getState()).toBe(true);
```

Reply:
changes done